vega/base.py

```diff
@@ -51,6 +53,8 @@ def _ipython_display_(self):
             metadata={'jupyter-vega': '#{0}'.format(id)}
         )
 
-    def display(self):
+    def display(self, renderer=None):
         """Render the visualization."""
+        if renderer:
```

Comment:
Default renderer is `canvas`.
I don't want that to be overwritten if the optional arg is entered.

But on 2nd thoughts, I could just have set it to `canvas` instead of `none`.

But I think I will remove the arg entirely, until I find a way to pass down the options without side effects